Automattic/wp-calypso
# [Cloud] Reset notification when trying to assign license

#### Proposed Changes

This PR aims to reset "Assign License" form errors so they no longer stack (see screenshot below). The reason behind this is that if consecutive, different, errors happens, it might unnecessarily confuse the user.

![Screenshot 2022-06-20 at 14 34 44](https://user-images.githubusercontent.com/3846700/175121471-71f14846-84fc-4e65-a5ab-1363d58f6545.png)

#### Context

* 1202074155672006-as-1202473762386419

#### Testing Instructions



* Spin up branch
* Spin up a JN site
* Connect your WPCOM account to the JN site
* Go to `http://jetpack.cloud.localhost:3000/partner-portal/licenses` and issue+assign a `Jetpack Complete` license to the JN site
* Go to `http://jetpack.cloud.localhost:3000/partner-portal/licenses` (again) and try to issue + assign a `Jetpack Anti-Spam` license to the JN site (or any other incompatible product)
* Press the `Assign to website` button a few times in a row and verify that the error is being reset/removed on each submission, so it only appears once
